src/createServer.js

```diff
@@ -1,3 +1,47 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { errorCase } = require('./Error');
+const { sendJSONResponse } = require('./senResponse');
```

Comment:
There is a typo in the import path for `sendJSONResponse`. It should be `./sendResponse` instead of `./senResponse` to match the file name.